packages/core/src/external/aws/lambda-logic/bundle-to-html-adhd.ts

```diff
@@ -0,0 +1,2339 @@
+    return dayjs(a.effectivePeriod?.start).isBefore(dayjs(b.effectivePeriod?.start)) ? 1 : -1;
+  });
+
+  const removeDuplicate = uniqWith(medicationsSortedByDate, (a, b) => {
+    const aDate = dayjs(a.effectivePeriod?.start).format(ISO_DATE);
+    const bDate = dayjs(b.effectivePeriod?.start).format(ISO_DATE);
+
+    return aDate === bDate && a.dosage?.[0]?.text === b.dosage?.[0]?.text;
+  }).filter(medicationStatement => {
+    const medicationDate = medicationStatement.effectivePeriod?.start ?? "";
+    const medicationDateFormatted = dayjs(medicationDate).format(ISO_DATE);
+    const aYearAgo = dayjs().subtract(1, "year").format(ISO_DATE);
+
+    return medicationDateFormatted > aYearAgo;
+  });
+
+  const otherMedications = removeDuplicate.filter(
+    medicationStatement => medicationStatement.status !== ("active" || "unknown")
+  );
+
+  const activeMedications = removeDuplicate.filter(
+    medicationStatement => medicationStatement.status === "active"
+  );
+
+  const emptyMedications = removeDuplicate.filter(
+    medicationStatement => !medicationStatement.status || medicationStatement.status === "unknown"
+  );
+
+  const activeMedicationsSection = createSectionInMedications(
+    mappedMedications,
+    activeMedications,
+    "Active Medications"
+  );
+
+  const emptyMedicationsSection = createSectionInMedications(
+    mappedMedications,
+    emptyMedications,
+    "Unknown Status Medications"
+  );
+
+  const completedMedicationsSection = createSectionInMedications(
+    mappedMedications,
+    otherMedications,
+    "Other Status Medications"
+  );
+
+  const medicalTableContents = `
+  ${activeMedicationsSection}
+  ${emptyMedicationsSection}
+    ${completedMedicationsSection}
+  `;
+
+  return createSection("Medications", medicalTableContents);
+}
+
+function getDateFormMedicationStatement(v: MedicationStatement): string | undefined {
+  return v.effectivePeriod?.start;
+}
+
+function createSectionInMedications(
+  mappedMedications: Record<string, Medication>,
+  medicationStatements: MedicationStatement[],
+  title: string
+) {
+  if (medicationStatements.length <= 0) {
+    const noMedFound = "No medication info found";
+    return ` <h4>${title}</h4><table><tbody><tr><td>${noMedFound}</td></tr></tbody></table>`;
+  }
+  const medicationStatementsSortedByDate = medicationStatements.sort((a, b) => {
+    const aDate = getDateFormMedicationStatement(a);
+    const bDate = getDateFormMedicationStatement(b);
+    if (!aDate && !bDate) return 0;
+    if (aDate && !bDate) return -1;
+    if (!aDate && bDate) return 1;
+    return dayjs(aDate).isBefore(dayjs(bDate)) ? 1 : -1;
+  });
+  const medicalTableContents = `
+      <h4>${title}</h4>
+      <table>
+    <thead>
+      <tr>
+        <th style="width: 25%">Medication</th>
+        <th style="width: 25%">Instructions</th>
+        <div style="width: 50%">
+          <th>Dosage</th>
+          <th>Code</th>
+          <th>Date</th>
```

Comment:
Remove “status” column in Medication, Encounters, Allergies, procedures, Immunizations, and Conditions sections.